internal/service/opensearchserverless/security_policy_data_source.go

```diff
@@ -0,0 +1,72 @@
+package opensearchserverless
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/errs/sdkdiag"
+)
+
+// @SDKDataSource("aws_opensearchserverless_security_policy")
+func DataSourceSecurityPolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadWithoutTimeout: dataSourceSecurityPolicyRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
The API has `createdDate` and `lastModifiedDate` in the response elements. What were your thoughts on omitting these from the DS?

Cite: https://docs.aws.amazon.com/opensearch-service/latest/ServerlessAPIReference/API_GetSecurityPolicy.html

Reply:
The Resource Contribution Guidelines suggested that timestamps should be skipped. Do you think they should still be included here?

> Skips Timestamp Attributes: Generally, creation and modification dates from the API should be omitted from the schema.

Cite: https://hashicorp.github.io/terraform-provider-aws/raising-a-pull-request/#resource-contribution-guidelines

Reply:
@joshjluo we do avoid timestamps in resources because of the high potential to cause continuous drift,  but they can be included in data sources